Makefile

```diff
@@ -55,11 +55,12 @@ ifdef USE_GPL
     LDLIBS  = -lgsl -lcblas
 endif
 
-prefix      = /usr/local
-exec_prefix = $(prefix)
-bindir      = $(exec_prefix)/bin
-mandir      = $(prefix)/share/man
-man1dir     = $(mandir)/man1
+prefix        = /usr/local
+exec_prefix   = $(prefix)
+bindir        = $(exec_prefix)/bin
+pkglibexecdir = $(exec_prefix)/libexec/bcftools
```

Comment:
You need to write a line in the Makefile,

    libexecdir = $(exec_prefix)/libexec

so that it can be overridden by the user with `make libexecdir=/somewhere/else`.  If you want, you can then define another variable in terms of it...